<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Parametro</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="agregarButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>file config</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>guardar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathFileEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>eliminar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
